v21.2/ui-statements-page.md

```diff
@@ -7,12 +7,28 @@ docs_area: reference.db_console
 
 {% include {{ page.version.version }}/ui/admin-access.md %}
 
-The **Statements** page helps you:
+The **Statements** page  of the DB Console helps you:
```

Comment:
this page exists on both DB and CC Console, so I don't think we should be saying only DB Console here (do we even use this term publicly?)

Reply:
Yes, we do use the term DB Console publicly: https://deploy-preview-13623--cockroachdb-docs.netlify.app/docs/dev/ui-overview.html

This particular page appears only in the CockroachDB docs: https://deploy-preview-13623--cockroachdb-docs.netlify.app/docs/dev/ui-statements-page.html

There's a different page for CC Console: https://deploy-preview-13623--cockroachdb-docs.netlify.app/docs/cockroachcloud/statements-page.html

The pages are constructed from shared components that are included into the parent page.